styles.css

```diff
@@ -1,4 +1,7 @@
 * {
+  /* number of Work Cards */
+  --numCards: 4;
```

Comment:
- [ ] `optional`
- [ ] Good job so far :clap: 
- [ ] Kindly add the blur background as the figma template to all the `desktop and mobile pop-up cards` please screenshots below
- `your template`
![Screenshot from 2022-07-22 00-01-33](https://user-images.githubusercontent.com/66230306/180322924-6e6e3fe5-67c4-4808-b2d9-533aec35c8ff.png)

- `figma template`
![Screenshot from 2022-07-21 20-00-11](https://user-images.githubusercontent.com/66230306/180323048-7e039f68-a129-4ed0-8092-4c0dd7f46066.png)